README.md

```diff
@@ -24,36 +25,382 @@ var level = require('level')
 //    This will create or open the underlying LevelDB store.
 var db = level('./mydb')
 
-// 2) put a key & value
+// 2) Put a key & value
 db.put('name', 'Level', function (err) {
   if (err) return console.log('Ooops!', err) // some kind of I/O error
 
-  // 3) fetch by key
+  // 3) Fetch by key
   db.get('name', function (err, value) {
     if (err) return console.log('Ooops!', err) // likely the key was not found
 
-    // ta da!
+    // Ta da!
     console.log('name=' + value)
   })
 })
 ```
 
-See **[LevelUP](https://github.com/rvagg/node-levelup)** and **[LevelDOWN](https://github.com/rvagg/node-leveldown)** for more details.
+<a name="api"></a>
+## API
+
+  * <a href="#ctor"><code><b>level()</b></code></a>
+  * <a href="#open"><code>db.<b>open()</b></code></a>
+  * <a href="#close"><code>db.<b>close()</b></code></a>
+  * <a href="#put"><code>db.<b>put()</b></code></a>
+  * <a href="#get"><code>db.<b>get()</b></code></a>
+  * <a href="#del"><code>db.<b>del()</b></code></a>
+  * <a href="#batch"><code>db.<b>batch()</b></code> *(array form)*</a>
+  * <a href="#batch_chained"><code>db.<b>batch()</b></code> *(chained form)*</a>
+  * <a href="#isOpen"><code>db.<b>isOpen()</b></code></a>
+  * <a href="#isClosed"><code>db.<b>isClosed()</b></code></a>
+  * <a href="#createReadStream"><code>db.<b>createReadStream()</b></code></a>
+  * <a href="#createKeyStream"><code>db.<b>createKeyStream()</b></code></a>
+  * <a href="#createValueStream"><code>db.<b>createValueStream()</b></code></a>
+
+See [`levelup`][levelup] and [`leveldown`][leveldown] for more details.
+
+<a name="ctor"></a>
+### `const db = level(location[, options[, callback]])`
+The main entry point for creating a new `levelup` instance.
+
+- `location` path to the underlying `LevelDB`.
+- `options` is passed on to the underlying store.
+- `options.keyEncoding` and `options.valueEncoding` are passed to [`level-codec`][level-codec], default encoding is `'utf8'`
```

Comment:
@ralphtheninja I missed this while reviewing. Why did you choose "passed to `level-codec`" instead of "passed to `encoding-down`"?

You could say `level-codec` has become an implementation detail of `encoding-down`. In which case I prefer "passed to `encoding-down`". But in the case of `level` you could also say that `encoding-down` is an implementation detail of `level` that adds `level-codec` support 😁

Reply:
Because it's actually level-codec taking care of the encodings. I'd be really happy if you could make a PR.

Reply:
PR incoming in a few minutes.